hnn_core/cell.py

```diff
@@ -4,12 +4,25 @@
 #          Sam Neymotin <samnemo@gmail.com>
 
 import numpy as np
+from numpy.linalg import norm
+
 from neuron import h, nrn
 
 # Units for e: mV
 # Units for gbar: S/cm^2
 
 
+def _get_yscale(p_secs, dipole_sec):
```

Comment:
See comment below, proposing to call this something to do with the projection that it is. It should be explicit that `dipole_sec` _defines_ the orientation. I'd prefer we don't use `yscale` anywhere, however...